src/Microsoft.AspNetCore.StaticFiles/CacheProfile.cs

```diff
@@ -0,0 +1,44 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.AspNetCore.StaticFiles
+{
+    /// <summary>
+    /// Defines settings used for caching responses for the <see cref="StaticFileMiddleware"/>.
+    /// </summary>
+    public class CacheProfile
+    {
+        /// <summary>
+        /// Gets or sets the duration for which the response is cached.
+        /// Sets the "max-age" in the "Cache-control" header in the 
+        /// <see cref="Microsoft.AspNetCore.Http.HttpContext.Response" />.
+        /// Defaults to 10 minutes
+        /// </summary>
+        public TimeSpan Duration { get; set; } = TimeSpan.FromMinutes(10);
```

Comment:
That's actually a good idea. Note that as per spec, negative numbers are invalid and should be ignored anyway.